app/core/models/ocelot/Page.scala

```diff
@@ -17,9 +17,23 @@
 package core.models.ocelot
 
 import core.models.ocelot.stanzas.Stanza
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
 
 case class KeyedStanza(key: String, stanza: Stanza)
 
+object KeyedStanza {
```

Comment:
With the extra functionality and tests it would be nice to moved keyedStanza to a dedicated file. Nice but not critical.

Reply:
KeyedStanza no longer used after serialisation update